css/team.css

```diff
@@ -0,0 +1,345 @@
+/*
+===================================
+===================================
+Our Team Styles
+===================================
+===================================
+*/
+
+/*
+===================================
+Member Card Component
+===================================
+*/
+/* member card style */
+.member {
+	background-color: var(--color-teal);
+	color: var(--text-color-light);
+	padding: var(--spacing-sm);
+	border-radius: var(--border-radius);
+	max-width: 654px;
+}
+/* img styling*/
+.member .img {
+	max-width: 240px;
+}
+.member img {
+	border-radius: var(--border-radius);
+}
+/* text */
+.member h2 {
+	font-size: 28px;
+	font-weight: 700;
+	line-height: 32px;
+}
+
+.member .role {
+	font-size: 24px;
+	font-weight: 500;
+	line-height: 28px;
+}
+.member p {
+	font-size: 16px;
+	font-weight: 400;
+	line-height: 20px;
+}
+
+/* member card layout */
+.member,
+.member .text,
+.member .bio,
+.member-cat {
+	display: flex;
+}
+
+/* similar gaps */
+.member,
+.member .text,
+.member-cat {
+	gap: var(--spacing-sm);
+}
+
+.member .bio {
+	gap: 10px;
+}
+
+/* text container and bio  */
+.member .text,
+.member .bio {
+	flex-direction: column;
+}
+.member .text {
+	justify-content: space-between;
+}
+
+/* team modal styles */
+.team-modal .member {
+	background: none;
+	width: 100%;
+	max-width: initial;
+	color: var(--text-color-dark);
+	padding: var(--spacing-sm) 0;
+}
+
+.team-modal .img {
+	max-width: 140px;
+}
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+
+.hero {
+	background: url("/assets/images/pages/team/hero-team.png"),
+		lightgray 50% / cover no-repeat;
+	display: flex;
+}
+
+/* flex layout */
+.hero .wrapper {
```

Comment:
There's another `.hero-wrapper` class that is being added in with "Events"

Reply:
sounds good, will remove for now and wait for events page pr to go up